qiskit/transpiler/passes/scheduling/alignments/releasenotes/notes/add-scheduler-warnings-ec87e3b3bcf654af.yaml

```diff
@@ -0,0 +1,65 @@
+---
```

Comment:
I think this file is duplicated with the one above. Also this is in the wrong directory, it should go in the `releasenotes/notes` directory off the root. I've made this mistake a lot of times when running `reno` outside the repository root.